indico/legacy/pdfinterface/conference.py

```diff
@@ -716,6 +661,43 @@ def _processDayEntries(self, day, story):
                         res.append(PageBreak())
         return res
 
+    def _print_contribution_with_abstract(self, res, contribution, sess_block):
+        title_line = contribution.title
+        if self._ttPDFFormat.showContribId():
+            title_line = f'[{contribution.friendly_id}] {title_line}'
+        if sess_block.session.is_poster and contribution.board_number:
+            title_line += f' (board {contribution.board_number})'
+        if hasattr(contribution, 'start_dt') and not sess_block.session.is_poster:
+            title_line += f' ({format_time(contribution.start_dt, timezone=self._tz)}'
+            if self._ttPDFFormat.showLengthContribs():
+                title_line += f', {format_human_timedelta(contribution.timetable_entry.duration)})'
+            else:
+                title_line += ')'
```

Comment:
Using a list and appending/prepending the parts would be nicer (and then joining the parts with an empty separator at the end). But since this is pretty much legacy code I'll leave it up to you whether you want to change it or not...

Reply:
Of course! Change done.